### PR TITLE
Snagging: Chat

### PR DESCRIPTION
**Pull request by sunny-viktoryia**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1817

image

### CODE REVIEW

File: src/components/attendee/Input/Input.tsx [hunk 1]
```diff
@@ -69,6 +102,8 @@ export const Input: React.ForwardRefRenderFunction<
           onKeyDown={handleKeyPress}
         />
 
+        {icon && renderIcon(icon, onIconClick)}
```
Comment (denisdimitrov) [inline src/components/attendee/Input/Input.tsx:108]: [NB] Some of the designs I've seen have this icon before the input on the left side, is there an easy way to do this or should we mark it OOS?

File: src/components/attendee/ChatContainer.module.scss [hunk 2]
```diff
@@ -100,11 +100,15 @@
   margin-bottom: space.empty(3);
 
   @media (min-width: breakpoint.$md) {
-    min-height: 90vh;
+    min-height: calc(100vh - 69px);
```
Comment (denisdimitrov) [inline src/components/attendee/ChatContainer.module.scss]: [NB] Can we use a constant for this 69px value? Is it the height of the footer or something else?

**Review by denisdimitrov — APPROVED** [review]
LGTM!

File: src/components/attendee/Input/Input.tsx (hunk 1 above)
Comment (sunny-viktoryia) [reply src/components/attendee/Input/Input.tsx:108]: I'd leave it for the next PR. 🙃

File: src/components/attendee/ChatContainer.module.scss (hunk 2 above)
Comment (sunny-viktoryia) [reply src/components/attendee/ChatContainer.module.scss]: added more meaning to the calculations here with the comment: https://github.com/sparkletown/sparkle/pull/3038/commits/ac1c1c2c500155a189e76ade05363d5799e129be

**Pull request merged**